backends/dpdk/dpdkProgram.cpp

```diff
@@ -483,6 +523,7 @@ boost::optional<int> ConvertToDpdkControl::getNumberFromProperty(const IR::P4Tab
 
 
 bool ConvertToDpdkControl::preorder(const IR::P4Table *t) {
+    checkTableValid(t);
```

Comment:
Are you OK not returning here on error?
You could have checkTableValid return a bool.

Reply:
Updated checkTableValid to return bool